configure.py

```diff
@@ -204,8 +204,8 @@ def create_build_configuration():
     write("build:windows --host_copt=/experimental:preprocessor")
     write("build:windows --copt=/arch=AVX")
 
-  if is_macos() or is_linux():
-    write("build --copt=-mavx")
+  # if is_macos() or is_linux():
```

Comment:
Don't remove directly, please add jugement, like this: https://github.com/tensorflow/addons/blob/33727beb4d05de93abeced03d25fe5668c78f5c0/configure.py#L143